vendor-protobufs/kuadrant/v1/descriptor_service.proto

```
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package kuadrant.v1;
```

Buf (1.67.0)

[error] Files with package "kuadrant.v1" must be within a directory "kuadrant/v1" relative to root but were in directory "vendor-protobufs/kuadrant/v1".

(PACKAGE_DIRECTORY_MATCH)